safe/impact_functions/core.py

```diff
@@ -143,3 +143,21 @@ def has_no_data(layer_data):
     :rtype: bool
     """
     return numpy.isnan(numpy.sum(layer_data))
+
+
+def get_key_for_value(value, value_map):
+    """Obtain the key of a value from a value map.
+
+    :param value: The value mapped to a key in value_map.
+    :type value: int, str, float
+
+    :param value_map: A value mapping.
+    :type value_map: dict
+
+    :returns: A key for the value.
+    :rtype: str
+    """
+    for key, values in value_map.iteritems():
```

Comment:
What happens if more than one key stores the same value. In the docstring at least we should mention that the first match is returned...